README.md

```diff
@@ -1,2 +1,131 @@
-# python-ffmpeg-feedstock
```

Comment:
https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Creating_new_packages.md#build-a-new-repository-from-scratch

at the end of this section, where the README.md is described, it is not mentioned that we need to copy the readme from the upstream project, but rather create the equivalent of the `about` section in the recipe.

Reply:
true, let me fix that. Thanks!